test_communication/test/message_fixtures.py

```diff
@@ -98,17 +98,18 @@ def get_msg_primitives():
     msg.bool_value = True
     msg.byte_value = bytes([1])
     msg.char_value = '\x01'
-    msg.float32_value = float(1.125)
-    msg.float64_value = float(1.125)
-    msg.int8_value = int(1)
-    msg.uint8_value = int(1)
-    msg.int16_value = int(1)
-    msg.uint16_value = int(1)
-    msg.int32_value = int(1)
-    msg.uint32_value = int(1)
-    msg.int64_value = int(1)
-    msg.uint64_value = int(1)
+    msg.float32_value = float(1.0)
+    msg.float64_value = float(1.0)
```

Comment:
Thanks for clarifying. This is only one of the four values. And the other ones cover the value range already. Sounds good to me as is then.